localstack/http/request.py

```diff
@@ -196,7 +196,12 @@ def get_raw_path(request) -> str:
     if hasattr(request, "environ"):
         # werkzeug/flask request (already a string, and contains the query part)
         # we need to parse it, because the RAW_URI can contain a full URL if it is specified in the HTTP request
-        return urlparse(request.environ.get("RAW_URI", request.path)).path
+        request_uri = request.environ.get("RAW_URI")
+        if request_uri:
+            return urlparse(request_uri.replace("//", "/")).path
+        return request.path
```

Comment:
@alexrashed could you sanity check this?

Reply:
I'd like to test that with S3 as well, it makes heavy use of `RAW_URI`

Reply:
(I think we have `test_resource_object_with_slashes_in_key` 
Which does requests like this which works:
`GET s3.localhost.localstack.cloud:4566/test-bucket-fa62efc6//bar/foo/`

Could update it to also run on virtual host to validate this format:
`GET test-bucket.s3.localhost.localstack.cloud:4566//bar/foo/`

But I think this PR is gonna make S3 tests fail and would break behavior.

Reply:
I have something up that I could thoroughly test with S3, should I open a PR? is it okay @calvernaz?

Reply:
Yep, let's do that! Will remove the changes on the `http` package

Reply:
The other PR has been merged, should be good to go to rebase!